test/kata4.humanCatDogYears.test.js

```diff
@@ -1,3 +1,13 @@
 const { humanCatDogYears } = require("../src");
 
-// Look Ma, no handlebars!!!
+describe("humanCatDogYears", () => {
+  it("returns an array of three numbers, the human years, the equivalent cat years and dog years", () => {
+    expect(humanCatDogYears(10)).toEqual([10, 56, 64]);
+  });
+
+  it("returns an array of three numbers, the human years, the equivalent cat years and dog years", () => {
+    expect(humanCatDogYears(20)).toEqual([20, 96, 114]);
+  });
+});
```

Comment:
I would put both these assertions together, as you are testing for the same thing thing in effect (passing in non zero values gives you non zero results). You could add another test to assert you get all zeroes back in the event of passing in zero, which is a valid case thats different enough to this set to be worthy of it (as exposes a but on the code as it happens):
```
describe("humanCatDogYears", () => {
  it("returns an array of three numbers, the human years, the equivalent cat years and dog years", () => {
    expect(humanCatDogYears(10)).toEqual([10, 56, 64]);
    expect(humanCatDogYears(20)).toEqual([20, 96, 114]);
  });

  it("returns an age of 0 for all species when passing in 0", () => {
    expect(humanCatDogYears(0)).toEqual([0, 0, 0]);
  });
});
```